release-pipeline.yml

```diff
@@ -1546,10 +1547,11 @@ stages:
           - name: Publish_To_Container
             value: ${{ parameters.publish_to_container }}
           - name: work_dir
-            value: '$(System.DefaultWorkingDirectory)/azure-storage-azcopy'
-
+            value: '$(Build.SourcesDirectory)'
 
         steps:
+          - checkout: self
```

Comment:
Do we need to checkout the code in test artifact job?

Reply:
Yes, we will need it as we are creating a drop.zip in this stage, where we also require NOTICE.txt from the repo to be copied into drop.zip